### DISCUSSION TITLE
Expose StyleSheet.hairlineWidth

### DISCUSSION THREAD

**Discussion by efstathiosntonas** [jpudysz/react-native-unistyles#164] [Feature Requests]
Ofcourse we can add it on our themes but it would be nice to be exposed (in runtime maybe?)

Long shot request but nice to have imho. It could be extended too and pass it a value instead of default `0.4`

```tsx
﻿﻿let hairlineWidth: number = PixelRatio.roundToNearestPixel(0.4);
if (hairlineWidth === 0) {
  hairlineWidth = 1 / PixelRatio.get();
}
```

**Comment by efstathiosntonas** [comment]
On top of that, we could expose `StyleSheet.absoluteFillObject` too

**Comment by efstathiosntonas** [comment]
in other words, get rid of `Stylesheet` everywhere 🤣

**Reply by jpudysz** [reply]
I was thinking about replacing StyleSheet with StyleSheet exported from Unistyles. It will be possible (I guess) without useStyles hook. I'm still trying to scratch cool architecture to achieve it. Stay tuned for 3.0!

**Comment by jpudysz** [comment]
Implemented in https://github.com/jpudysz/react-native-unistyles/pull/250 (2.9.0)
